### DISCUSSION TITLE
Crash when trying to time using T key while the first timing point is not selected

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#21385] [Q&A] [answered]
Pressing T to tap around the timing point without it being selected causes unhandled errors the crashes.

![image](https://user-images.githubusercontent.com/53081750/203485064-34757dde-a695-4dbf-b2ec-25b37d7d6407.png)

[runtime.log](https://github.com/ppy/osu/files/10073195/runtime.log)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/21386
